bamboo/unit_tests/test_unit_mnist_softmax_classifier.py

```diff
@@ -18,7 +18,7 @@ def skeleton_mnist_softmax_classifier(cluster, executables, dir_name, compiler_n
         num_processes=1, dir_name=dir_name,
         data_filedir_default='/p/lscratchh/brainusr/datasets/MNIST',
         data_reader_name='mnist',
-        data_reader_percent=data_reader_percent,
+        data_reader_percent=0.01,
```

Comment:
Same question as above.  Is this overridden by the normal weekly / nightly percentage.

Reply:
This change will be reverted and then this test is moving to `weekly`-only. As @timmoon10 noted, it's "parameter bound", not "data bound" so changing this value actually didn't affect anything.